.gitattributes

```diff
@@ -0,0 +1,61 @@
+# Drupal git normalization
```

Comment:
What was the motivation for making this a non-scaffolded file? Is it thought that frequently the user will have the need to modify it?

Reply:
If it is scaffolded, it is a build artifact. If it is a build artifact, it is not in the working tree when developers clone an IC repo in the usual way. But this file's audience / value is those developers.